README.markdown

```diff
@@ -266,6 +266,10 @@ Rsync comment.
 
 File containing motd info.
 
+##### `pid_file`
+
+PID file. Defaults to /var/run/rsyncd.conf.
```

Comment:
Not sure if you mean "/var/run/rsyncd.pid"?

Reply:
actually it's not an option for the ::module definition but for the ::server class only, because it's global so I am not sure if this is the right place where to add documentation for it ... documentation for rsync::server seems to be missing completely